docs/analyzer/user_guide.md

```diff
@@ -2579,3 +2593,30 @@ void testError2() {
   int x = 1 / 0;
 }
 ```
+
+### Setting with config file
+
+Review status can be configured by a config file in YAML format. This config
+file has to represent a list of review status settings:
+
+```yaml
+- filepath: /path/to/project/test/**
+  checker: core.DivideZero
+  message: Division by zero in test files is automatically intentional.
+  review_status: intentional
+- filepath: /path/to/project/important/module/**
+  message: All reports in this module should be investigated.
+  review_status: confirmed
+```
+
+The fields of a review status settings are:
+
+- `filepath` (optional): A glob to a path where the given review status is
+  applied.
+- `checker` (optional): Set the review status for only these checkers' reports.
+- `message` (optional): A comment message that describes the reason of the
+  setting.
+- `review_status`: The review status to set.
```

Comment:
What are the possible values?

Is ignore supported?

Reply:
Now I validate the possible values. `ignore` is not supported yet.

Reply:
Please create a follow-up issue for this.